Documentation/network/concepts/ipam/eni.rst

```diff
@@ -87,6 +87,28 @@ Configuration
   You may override this behavior by setting a cluster-specific ``--eni-gc-tags``
   tag set.
 
+* The following IPAM parameters can be configured via Helm values
+  or operator flags to control IP address allocation behavior:
+
+  * ``ipam.operator.preAllocate`` (Helm) or ``--ipam-pre-allocate`` (operator flag):
```

Comment:
Might be worth mentioning these in the upgrade notes (i.e. `Documentation/operations/upgrade.rst`).